src/deletepy/__init__.py

```
@@ -0,0 +1,209 @@
+    BASE_RETRY_DELAY,
+    MAX_RETRIES,
+    MAX_RETRY_DELAY,
+    get_base_url,
+    get_env_config,
+    get_estimated_processing_time,
+    get_optimal_batch_size,
+    validate_rate_limit_config,
+)
+
+# Operations
+from .operations.batch_ops import (
```

Flake8 (7.2.0)

[error] '.operations.batch_ops._categorize_users' imported but unused

(F401)